src/status_im2/contexts/chat/photo_selector/album_selector/view.cljs

```diff
@@ -41,7 +42,7 @@
 
 (defn section-header
   [{:keys [title]}]
-  (when (not= title "smart-albums")
+  (when-not (= title "no-title")
```

Comment:
where are we getting this string "no-title" from, should it not be in some common vars or something like that?

Reply:
why rename it to "no-title"? The API that fetches these albums is called "smart albums"

Reply:
we don't use this API, and its not clear why we want to hide title for smart-albums, so now its clear if no-title we just want to hide-title , so its just more general implementation

Reply:
It's fine just a nitpick. 

> we don't use this API
We don't use it yet, but we have to in order to get the favorites album. Yet to be implemented.